language/German/strings.po

```diff
@@ -3098,7 +3148,7 @@ msgstr "Unterstützt: Ja"
 
 #: src/frontend/mame/ui/selmenu.cpp:681
 msgid "System is parent"
-msgstr ""
+msgstr "System ist Elternteil"
```

Comment:
"Elternteil"? who would actually understand what that means :-D 
how about "übergeordnetes System" or "Basissystem" or something like that?

Reply:
I don't, but had no better idea, I like _übergeordnetes System_
Changed